### PR TITLE
login issues: Updated with current information

### PR DESCRIPTION
**Pull request by garrett**
- Removed more distribution-specific instructions
- Added Rocky and Alma to the fixed paragraph
- Reworded sed command
- Added a mention about browser caching

### CODE REVIEW

**Comment by garrett** [comment]
Preview: https://garrett.github.io/cockpit-project.github.io/blog/login-issues.html

**Comment by garrett** [comment]
Oh, I should add an update at the top like you did, @martinpitt.

File: _posts/2022-10-20-login-issues.md [hunk 1]
```diff
@@ -8,13 +8,14 @@ tags: cockpit
 
 *Updated on 2022-10-25*: Fix for Ubuntu 22.04 LTS becomes available as official update.
```
Comment (garrett) [inline _posts/2022-10-20-login-issues.md:9]:
```suggestion
**_Last updated on 2022-11-09**: All major distributions should now have fixed packages._
```

**Review by martinpitt — APPROVED** [review]
Danke!

**Pull request merged**